scripts/readme-script.ts

```diff
@@ -0,0 +1,54 @@
+import { COMMANDS } from '../src/commands/commands';
+import { promises } from 'fs';
+import { stripIndents } from 'common-tags';
+import chalk from 'chalk';
+const { readFile, writeFile } = promises;
+
+/**
+ * Available Flags:
+ * • --updateReadme (This flag will make the script update the readme)
+ */
+const flags = process.argv.slice(2);
+
+const commandTable = stripIndents`
+  ## Available Commands
+  | Prefix | Description |
+  | :-: | :-: |
+  ${COMMANDS.map(
+    (command) => '| ' + command.prefix + ' | ' + command.description + ' |\n'
+  ).join('')}
+`;
+
+// if there is an argument --autoUpdateReadme while running the command,
+// it will run this
+if (flags.includes('--autoUpdateReadme')) {
+  readFile('README.md', 'utf-8')
+    .then((data) => {
+      // Change the regex in this line to determine where you're going to put it at...
+      const results = data.replace(
+        /## Available Commands.*## Additional Information/gs,
+        commandTable
+      );
+
+      writeFile('README.md', results, 'utf8')
```

Comment:
utf8 is different than the "readFile` call of `utf-8` is that ok? Or just inconsistent

Reply:
That is ok

Reply:
I will just edit it anyway